spec/config.spec.ts

```diff
@@ -58,6 +58,6 @@ describe('config()', () => {
 
   it('throws an error if Firebase configs not present', () => {
     mockRequire('../../../.runtimeconfig.json', {});
-    expect(config).to.throw('Firebase config variables are missing.');
+    expect(config).to.throw('Firebase config variables are not available.');
```

Comment:
We added more complex behaviors. Could we test them? Hint: I'm pretty sure I see a bug =p